### ISSUE TITLE
where does the plugin install the models to

### ISSUE THREAD

**Issue by Hicham-Nait-Bakkou** [jpmcb/nvim-llama#12]
i want to uninstall a model of ollama so i can install another one
thx

**Comment by adventurist** [comment]
Indeed, I'd love to know that without digging.. but I suppose it's going to be into the docker environment, and that it'll be system default
